### ISSUE TITLE
Scss incompatible with standard tools (references to undeclared mixins)

### ISSUE THREAD

**Issue by werrolf** [mapbender/mapbender#761]
Mapbender's asset pipeline uses some highly custom machinery, including eslider/sasscb, which is a bunch of prebuilt binaries built with unknown options on an unknown host system.

These binaries seem to be somewhat lenient against errors for unknown reasons.

The standard tool for compiling scss is [ruby sass](https://sass-lang.com/install), but the current Mapbender template scss files are incompatible with it and throw errors.

```
ron@flake:~/proj/mb305/application$ sass --scss \
  -Imapbender/src/Mapbender/CoreBundle/Resources/public/sass \
  -Ivendor/igosuki/compass-mixins/lib \
  -Iweb
  mapbender/src/Mapbender/CoreBundle/Resources/public/sass/template/fullscreen.scss
Error: ".mb-element-popup-dialog" failed to @extend ".clearfix".
       The selector ".clearfix" was not found.
       Use "@extend .clearfix !optional" if the extend should be able to fail.
        on line 45 of /home/ron/proj/mb305/application/mapbender/src/Mapbender/CoreBundle/Resources/public/sass/modules/_popup_dialog.scss
  Use --trace for backtrace.
```
(include paths plucked from and equivalent to [current Mapbender mapbender.assetic.filter.sass configuration](https://github.com/mapbender/mapbender/blob/fdabe71d1ac8d9a94cc5c565d42bbbd3b7e09f9b/src/Mapbender/CoreBundle/Resources/config/services.xml#L68))

There are equivalent errors for [all the other templates](https://github.com/mapbender/mapbender/tree/fdabe71d1ac8d9a94cc5c565d42bbbd3b7e09f9b/src/Mapbender/CoreBundle/Resources/public/sass/template).

The issue boils down to references to undeclared mixins `.clearfix` and `.ie-8-borders-hack`. These may at some point have been defined, but no longer appear in the codebase / installed dependencies.

There actually still exists a [%clearfix placeholder selector without the dot](https://github.com/mapbender/mapbender/blob/fdabe71d1ac8d9a94cc5c565d42bbbd3b7e09f9b/src/Mapbender/CoreBundle/Resources/public/sass/libs/_mixins.scss#L106) . It's just not used at all in the templates.
(see [here for a explanation of placeholder-only mixins](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#placeholder_selectors_foo))

Removing these mixin references resolves the erros, and the door is open to compiling the stylesheets with standard tools.

**Comment by werrolf** [comment]
```
ron@flake:~$ sass --version
Sass 3.4.21 (Selective Steve)
ron@flake:~$ dpkg -S `which sass`
ruby-sass: /usr/bin/sass
ron@flake:~$ cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=16.04
DISTRIB_CODENAME=xenial
DISTRIB_DESCRIPTION="Ubuntu 16.04.3 LTS"
```

**Comment by werrolf** [comment]
Fixed in commit c01ea7e4b5be4466b08dd1c789524813112ecf32.
